docs/user/guides/build/email-notifications.rst

```diff
@@ -0,0 +1,40 @@
+How to enable email notifications
+=================================
+
+In this guide, you can learn how to setup build notification via email.
+
+.. note::
+
+   Currently we don't send email notifications on :doc:`builds from pull requests </pull-requests>`.
```

Comment:
Hmm.. this seems like an odd limitation, since that seems like the most useful time for them! I think this was originally defensive, but likely should be enabled (cc @humitos, do you remember why this was done?)

Reply:
IIRC it is because it would be a flood of emails. I personally don't see email notifications on PR super useful. When you are working on a PR you will probably see the GitHub check on the PR's page and that's probably enough, so I'd say there is no need for an email notification.

Also note that the notifications go out to all the emails subscribed to, so people will receive emails about failed builds where they are not even involved in those PRs. IMO, the decision of not sending emails on PR is correct 😄

Reply:
In the future, we could probably make each type of notification opt-in; like "I want to receive all notifications" or "I want to receive notifications on regular versions but not on PRs".

I don't see a ton of value here either 🙃

Reply:
I agree with @humitos 

It's funny though.. because Circle CI sends notifications about PR failures... but in this case, I find it spammy since I already monitor my own PRs. They don't provide an obvious way to turn it off.

But it's also super-different from how we do it: I think they use Git commit info to match email addresses with notification settings.

So very complicated and not sure that the end-result is less easy for a user:

![image](https://github.com/readthedocs/readthedocs.org/assets/374612/ec4dc9ed-d518-4ebd-ac78-4c334b992aad)

Reply:
Noting that the notes are moved down to a location where they are more in context and less noisy. I also trimmed out the word "Currently" - I wasn't sure what it was meant to do, it could be indicating an upcoming change in behavior or otherwise just cast doubt on functionality.

Reply:
Yes, but actually it's a bit the opposite that I think we have prioritized... we DON'T want notifications about user's own PR builds.. but I can see how extremely noisy it would be once adding PRs that the user isn't even part of :)

For Circle CI, the use case is also different... CI builds are often slow and the user will switch context (close the browser tab) and use email notifications to re-engage later.

I think if we enabled email notifications for PR builds, we would immediately have users requesting a way to turn it off. However, _not_ having it, doesn't seem to have made anyone request it :shrug: